src/common/queries/account.ts

```diff
@@ -34,7 +37,9 @@ export const getInstalledModules = async ({
   const responseBody = await response.json()
 
   if (response.ok) {
-    return responseBody.data.moduleQueries.map((module: any) => module.module)
+    return responseBody.data.SmartAccount_ModuleQuery.map(
+      (module: any) => module.module,
```

Comment:
this is kind of unrelated but it might make sense to rename the `module` field to `moduleAddress` - we can do this now or a bit later too

Reply:
yeah not a problem here I mean generally